geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/process/query/QueryProcess.scala

```diff
@@ -98,6 +105,9 @@ class QueryVisitor(features: SimpleFeatureCollection,
     source.getFeatures(query)
   }
 
+  override def getExpressions: java.util.List[Expression] = {
+    FilterHelper.propertyNames(filter, origSft).map(ff.property).toList
```

Comment:
If `propertyNames` works correctly with the Include filter than this is return all the attributes but if we include properties here I think we could end up forcing a retyping feature collection. Properties can rename and create attributes which if given I believe would cause a retyping?